### ISSUE TITLE
✅ 🐞 Save Button Not Showing In Conditions Dialogue Box for Custom Code instances

### ISSUE THREAD

**Issue by ammar14z** [elementor/elementor#28138]
### Prerequisites

- [X] I have searched for similar issues in open and closed tickets and cannot find a duplicate.
- [X] I have troubleshooted my issue, and it still exists against the latest stable version of Elementor.

### Description

After new update of Elementor Pro (Version 3.24.0-cloud4), the save button in conditions dialogue box of adding new Custom Code  is disappeared. Which makes this custom code feature useless because we cannot assign custom code to any page, post or whatsoever. I have disabled all the other plugins except for Elementor & Elementor pro. It only appears when I rolled back Elementor pro to version 3.24.0-cloud3. After rolling back save button appears. For your reference I am attaching first photo with latest version and second photo with old version.

![No-Save-Button](https://github.com/user-attachments/assets/2bd010be-4f6b-4161-bab5-1f47fcef4192)
![Old-Save-Button](https://github.com/user-attachments/assets/df086da4-87b5-48ca-844e-5bc6cd4d41f1)

### Steps to reproduce

1. Click On Elementor > Custom Code > Add New Custom Code
2. Write Any Title & Dummy Custom Code
3. Click On Publish
4. In Display Conditions Dialogue Box you can set any display condition but no way to save that condition because no save button disappeared from the dialogue box.

### Expected behavior

It supposed to show the save button in the dialogue box.

### Isolating the problem

- [x] This bug happens when only the Elementor (and Elementor Pro) plugins are active.
- [x] This bug happens with the Hello Elementor theme active.
- [x] I can reproduce this bug consistently by following the steps I described above.

### Elementor System Info

<details>
<summary>Click to reveal</summary>

````txt
== Server Environment ==
	Operating System: Linux
	Software: nginx/1.24.0
	MySQL version: (Google) v8.0.31
	PHP Version: 7.4.33
	PHP Memory Limit: 512M
	PHP Max Input Vars: 10000
	PHP Max Post Size: 256M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.6
	Site URL: https://coreapplet.com
	Home URL: https://coreapplet.com
	WP Multisite: No
	Max Upload Size: 256 MB
	Memory limit: 512M
	Max Memory limit: 512M
	Permalink Structure: /%postname%/
	Language: en_US
	Timezone: 0
	Debug Mode: Inactive

== Theme ==
	Name: Hello Elementor
	Version: 3.1.0
	Author: Elementor Team
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: en-US
	User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/126.0.0.0 Safari/537.36

== Active Plugins ==
	Elementor
		Version: 3.24.0-cloud5
		Author: Elementor.com

	Elementor Pro
		Version: 3.24.0-cloud4
		Author: Elementor.com


== Elements Usage ==
	


== Settings ==
	
	allow_tracking: yes
	font_display: swap


== Features ==
	Custom Fonts: 0
	Custom Icons: 0

== Integrations ==
	


== Elementor Experiments ==
	Improved CSS Loading: Active by default
	Inline Font Icons: Active
	Additional Custom Breakpoints: Active by default
	Flexbox Container: Active by default
	Grid Container: Active by default
	Upgrade Swiper Library: Active by default
	Nested Elements Performance: Active by default
	Optimized Control Loading: Active by default
	Optimized Markup: Inactive by default
	Plugin Onboarding: Active by default
	Default to New Theme Builder: Active by default
	Header & Footer: Active by default
	Elementor Home Screen: Inactive
	Build with AI: Active by default
	Landing Pages: Inactive by default
	Nested Elements: Active
	Lazy Load Background Images: Active by default
	Element Caching: Active by default
	Link In Bio: Active by default
	Floating Buttons: Active by default
	Elementor Editor Events: Active
	Atomic Widgets: Inactive by default
	Editor Top Bar: Active
	Floating Bars: Inactive by default
	Improved Page Actions Layout: Active
	Display Conditions: Active by default
	Form Submissions: Active by default
	Menu: Inactive by default
	Taxonomy Filter: Active by default
	Search: Active by default


== Log ==
	
Log: showing 5 of 52024-07-19 13:43:36 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.24.0-cloud4',
  'to' => '3.24.0-cloud5',
)]
2024-07-19 13:43:47 [info] elementor::elementor_updater Started 
2024-07-19 13:43:47 [info] Elementor/Upgrades - _on_each_version Start  
2024-07-19 13:43:47 [info] Elementor/Upgrades - _on_each_version Finished 
2024-07-19 13:43:48 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.24.0-cloud4',
  'to' => '3.24.0-cloud5',
)]



== Elementor - Compatibility Tag ==
	
	Elementor Pro: Incompatible

== Elementor Pro - Compatibility Tag ==

````

</details>

### Agreement

- [X] I agree that my issue may be closed without action if it doesn't meet all the requirements.

**Comment by nicholaszein** [comment]
Hello @ammar14z! 

Thank you for submitting your issue! 🙏 

Please allow us some time to investigate it, and get back to you with our findings. 🧐 

Best regards

**Comment by nicholaszein** [comment]
Hi @ammar14z! 

Thank you for your patience! 🙏 

We are aware of this issue and our team is already working on a fix. We can’t say exactly when it'll be sorted out, but we kindly ask for your patience. Thanks for understanding! 😉

Kind regards

**Comment by nicholaszein** [comment]
Hey @ammar14z!

We have great news! 🙌 

### 📢 We're happy to announce that the issue you raised was resolved in Elementor Pro v3.24.0-cloud8! 🥳 

Thank you for sharing your feedback, and for helping us finding this issue!

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
